### ISSUE TITLE
Chain-B pdb file of 5qc6

### ISSUE THREAD

**Issue by ppxasjsm** [csimmerling/D3R-GC4#1]
Hi everyone, 

The repo looks very interesting. I was wondering if you had the pdb structure you started out with and put through tleap that matches the coordinates of the mol2 files. 
I think in the original submission you had to provide the ligand mol2 file and the corresponding pdb structure, something like that would be super helpful, rather than the already merged molecules. 

I also noticed that the intermediates A,B,C, and D seem to be shifted in comparison to the mol2 files given which seems a bit odd. 

Lastly, I noticed you modelled everything on the compound (BCJ) in 5qc6 of chain B. Is there a reason for this particular choice? Chain A seems to model the compounds slightly differently with the piperidine in chair conformation, versus in boat in chain B. I would expect it to be in chair rather than boat. Any thoughts on this? 

Best,
-Toni

**Comment by csimmerling** [comment]
Hi Toni,
Sorry for the delay. Thanks for your interest in this repo.
1. Please check the pdb folder in the updated repo. tleap will fill the missing atoms.
2. Mol2 files for substructures A,B,C&D are fixed.
3. Regarding the choice of chain in 5qc6,
In chain A and chain B of PDB ID 5qc6, the ligand BCJ differs in the conformation of one of the piperidine groups which involves atom N70. They also differ in the direction of the charge carrying proton attached to N70. Since interchanges between the chair and boat conformation of the two piperidine groups are frequently observed during the simulations, we believe that the initial conformation of the piperidine groups is a less concern. However, protonation of the N70 atom in the ligand BCJ of chain A will result in the charge carrying proton facing the protein and being buried, which is energetically unfavorable. In chain B, the charge carrying proton attached to N70 will form a pi-cation interaction with Phe70, which is more energetically favorable. 
Best.

**Comment by ppxasjsm** [comment]
Hi Carlos, 

Thank you for the pdb files, and your explanation above. I had a bit of a closer look at the protein structures and I noticed that  you have 7 residues labelled as CYX, rather than just the 6 that should be forming the disulphide bridges. I believe residue 12, should be a CYS. I am not sure if this has any serious effect on the simulation, but I am not sure how you managed to generate a valid topology, or at least I struggle to using CYX. However, the topolgoy files in the 7z archives appear to have also 7 CYX residues. Any thoughts on this?

**Comment by csimmerling** [comment]
yes you're right, residue 12 should be Cys and not Cyx. The student made a mistake when building it. It's quite far from the binding site so I don't think it had an impact during the short MD. Leap doesn't have an issue with an unpaired Cyx so there was no error. What error are you getting?
